src/main/resources/application.yml

```diff
@@ -66,6 +66,9 @@ exceptionmanager:
     host: localhost
     port: 8666
 
+schedule:
+  time: 0 16 * * * *
```

Comment:
This cron will also run on Saturdays and Sundays. Is that intentional?

Reply:
The Trello card talks about having it run on weekends so I left the `MON-FRI` out. I can double check just to be sure